### PR TITLE
Fix bug in docstring.

### PR DESCRIPTION
**Pull request by zytx121**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Update docstring.

## Modification
Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: mmrotate/models/dense_heads/kfiou_odm_refine_head.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 @ROTATED_HEADS.register_module()
 class KFIoUODMRefineHead(KFIoURRetinaHead):
-    """Rotational Anchor-based refine head.
+    """Rotated Anchor-based refine head for KFIoU.
```
Comment (ZwwWayne) [inline mmrotate/models/dense_heads/kfiou_odm_refine_head.py:13]: ODM seems to be not described.

File: mmrotate/models/dense_heads/kfiou_rotate_retina_head.py [hunk 2]
```diff
@@ -5,24 +5,18 @@
 
 @ROTATED_HEADS.register_module()
 class KFIoURRetinaHead(RotatedRetinaHead):
-    """Rotational Anchor-based refine head.
+    """Rotated Anchor-based head for KFIoU.
```
Comment (ZwwWayne) [inline mmrotate/models/dense_heads/kfiou_rotate_retina_head.py:8]: The summary should be more specific

File: mmrotate/core/bbox/assigners/sas_assigner.py [hunk 3]
```diff
@@ -43,9 +46,11 @@ def get_horizontal_bboxes(gt_rbboxes):
 
 
 def AspectRatio(gt_rbboxes):
-    """compute the aspect ratio of all gts
+    """compute the aspect ratio of all gts.
```
Comment (yangxue0827) [inline mmrotate/core/bbox/assigners/sas_assigner.py:49]: 首字母大写

File: mmrotate/models/losses/convex_giou_loss.py [hunk 4]
```diff
@@ -308,9 +312,11 @@ def forward(self,
 
 
 def AspectRatio(gt_rbboxes):
-    """compute the aspect ratio of all gts
+    """compute the aspect ratio of all gts.
```
Comment (yangxue0827) [inline mmrotate/models/losses/convex_giou_loss.py:315]: 首字母大写

File: tools/analysis_tools/benchmark.py [hunk 5]
```diff
@@ -61,18 +61,20 @@ def parse_args():
...
-            checkpoint (str): Checkpoint file path.
-            max_iter (int): Num of max iter.
-            log_interval (int): Interval of logging.
-            is_fuse_conv_bn (bool): Whether to fuse conv and bn,
-                this will slightly increase the inference speed
-            use_fp16 (bool): Whether to use fp16 to inference.
-        Returns:
-            fps (float): Average speed of inference (fps).
-        """
+    """inference speed statistics.
```
Comment (yangxue0827) [inline tools/analysis_tools/benchmark.py:64]: 首字母大写

File: tools/analysis_tools/benchmark.py [hunk 6]
```diff
@@ -155,21 +157,23 @@ def repeat_measure_inference_speed(cfg,
...
-            is_fuse_conv_bn (bool): Whether to fuse conv and bn,
-                this will slightly increase the inference speed
-            use_fp16 (bool): Whether to use fp16 to inference.
-            repeat_num (int): Number of repeat times of measurement
-                for averaging the results.
-        Returns:
-            fps (float of list(float)): Inference speed(fps) or
-                list of inference speed(fps) for repeating measurements.
-        """
+    """repeat to inference several times and take the average.
```
Comment (yangxue0827) [inline tools/analysis_tools/benchmark.py:160]: 首字母大写

**Review by yangxue0827 — APPROVED** [review]
LGTM

**Comment by ZwwWayne** [comment]
There are still some tweaks to be fixed
- [ ] in mmrotate.core.bbox.DeltaXYWHAOBBoxCoder
image
- [ ] mmrotate.core.bbox.GVFixCoder.decode
- [ ] mmrotate.models.dense_heads.KFIoUODMRefineHead.get_bboxes
image
- [ ] similar issue in mmrotate.models.dense_heads.KFIoURRetinaRefineHead, mmrotate.models.dense_heads.RotatedRPNHead,

**Pull request merged**